tests/conftest.py

```diff
@@ -40,6 +40,9 @@ def pytest_configure(config):
     config.addinivalue_line("markers", "target_pool: run test against one or more specific pool")
     config.addinivalue_line("markers", "skip_pool: exclude one or more pools in this test")
     config.addinivalue_line("markers", "skip_meta: exclude metapools in this test")
+    config.addinivalue_line("markers", "skip_a_rate: exclude aToken-style pools in this test")
+    config.addinivalue_line("markers", "skip_c_rate: exclude cToken-style pools in this test")
+    config.addinivalue_line("markers", "skip_eth: exclude ETH pools in this test")
```

Comment:
instead of unique decorators to skip each pool type, i feel like it'd be cleaner to have a single `skip_pool_type` decorator that works similarly to `skip_pool` and skips based on a list of pool types as strings:

```python
@pytest.mark.skip_pool_type("eth", "a_rate")
```